### ISSUE TITLE
Telnet Client shows as Deprecated

### ISSUE THREAD

**Issue by shawty** [9swampy/Telnet#81]
Trying to figure out, if this is something in my Visual Studio setup, of if there really is something in this telnet library causing it.

![image](https://github.com/user-attachments/assets/e5d2e510-97f2-4462-b248-6f1d86f7acfb)

I'm building a .NET standard 2.0 assembly (as it has to be used by .NET 4.8 clients as well as more modern dotnet X clients.

Working under VS2022 on a Win 10 x64 system, I've had a search of the issues already on here, and a look through the code base, can't see anything that would suggest it's deprecated.

Package was added to my project from the latest version available on NuGet.

Cheers
Shawty

**Comment by shawty** [comment]
PS: It all compiles and works ok, so it's not a build breaker or anything.

**Comment by 9swampy** [comment]
It's not a problem, per se, but as the message advises it would be better if you could refactor your code to use the overload of the constructor that accepts a TcpByteStream which you can dispose of correctly. Calling the overload that you are doesn't.

**Comment by 9swampy** [comment]
FWIW you prompted me to bump the Nugets as I've not done that for quite a while.... 0.13.1 otw

**Comment by shawty** [comment]
@9swampy - Thanks for that.  It had me quite confused, because I couldn't find the actual constructor with the deprecated notice added to it, so I was wondering if it was maybe coming from some other part of the .NET stack.

and... don't even get me started on what co-pilot for Visual Studio thought :-)

I did eventually re-factor things anyway, as my library required Connect/Disconnect API entry points to be separated, so all good.

Thanks for the clarification, and glad I provided you with a reminder :-)
